### ISSUE TITLE
prov/verbs,prov/rxm: failure in fi_poll -t counter -p "verbs;ofi_rxm" for branch master

### ISSUE THREAD

**Issue by jswaro** [ofiwg/libfabric#4548]
```
crayadm@uan1:/scratch/jswaro/libfabric/contrib/cray> bin/run_libfabric_pipeline -w /scratch/jswaro/workspace -l /scratch/jenkins/builds/libfabric/tags/nightly -v -s fabtest | tee log
...

##############################################################################
Start of "test"
##############################################################################


##############################################################################
Start of "fabtests"
##############################################################################

- name:   fi_getinfo_test -s 10.0.0.1 10.0.0.1 -p "verbs;ofi_rxm"
  result: Pass
  time:   1
...
- name:   fi_poll -t counter -p "verbs;ofi_rxm"
  result: Fail
  time:   1
  server_cmd: /scratch/jswaro/workspace/fabtests/bin/fi_poll -t counter -p "verbs;ofi_rxm" -s 10.0.0.1
  server_stdout: |
    Posting a send...
    Retrieved 2 event(s)
    Send counter poll-event
    Invalid tx counter event
  client_cmd: /scratch/jswaro/workspace/fabtests/bin/fi_poll -t counter -p "verbs;ofi_rxm" -s 10.0.0.2 10.0.0.1
  client_stdout: |
    Posting a send...
    Retrieved 2 event(s)
    Send counter poll-event
    Invalid tx counter event
- name:   fi_rdm -p "verbs;ofi_rxm"
  result: Pass
  time:   2
...
- name:   fi_dgram_pingpong -I 5 -p "verbs;ofi_rxm"
  result: Notrun
  time:   2
# ------------------------------------------------------------------------------
# Total Pass                                                31
# Total Notrun/Excluded                                     39
# Total Fail                                                 1
# Percentage of Pass                                        96
# ------------------------------------------------------------------------------
srun: error: cn1: task 0: Exited with exit code 1

##############################################################################
End of "fabtests"
##############################################################################
```

**Comment by a-ilango** [comment]
Thanks. I've opened a PR that fixes the issue.

**Comment by jswaro** [comment]
closed by #4552
